scripts/windows_builder/check_and_build.py

```diff
@@ -49,7 +49,7 @@
 base_dir = "C:\Users\Jeff Reback\Documents\GitHub\pandas"
 remote_host='pandas.pydata.org'
 username='pandas'
```

Comment:
remove the entire directory: scripts/windows_builder as these are not longer used